plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/io/HttpUtil.java

```diff
@@ -31,6 +41,37 @@
   public static final int DEFAULT_CONNECT_TIMEOUT_MS = 3000;
   public static final int DEFAULT_READ_TIMEOUT_MS = 3000;
 
+  private static final String MULTIPART_BOUNDARY =
```

Comment:
It's some arbitrary UUID I generated. It's perfectly fine to reuse the same boundary every time, as long as it gets its job done to separate multiple parts (i.e., it's a unique boundary value that never appears in any actual parameter values).